Tests/iaas/flavor-naming/flavor-form.py

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env python3
+# Little wrapper to output flavor pretty description
+# from a CGI form with flavor
+#
+# (c) Kurt Garloff <kurt@garloff.de>, 11/2023
+# SPDX-License-Identifier: CC-BY-SA-4.0
+"""
+flavor-form.py
+CGI script to get passed flavor from a html form (GET)
+and parses it according to SCS flavor naming.
+It returns an error (sometimes with a useful error message)
+or a human-readable description of the flavor.
+"""
+
+# import os
+import sys
+import re
+# import traceback
```

Comment:
Shall we get rid of the imports that are commented out?

Reply:
Those were required before but have been replaced ...
Feel free to push a change to remove them.